package.json

```diff
@@ -108,19 +108,16 @@
     "angular-cookies": "^1.7.9",
     "angular-route": "^1.7.9",
     "bootstrap": "^3.4.1",
-    "create-react-class": "^15.6.2",
     "cropperjs": "^1.5.6",
     "default-passive-events": "^1.0.10",
+    "fabric": "^3.6.3",
```

Comment:
How much do these changes affect the total amount of data? Preferably gziped and minified.

Reply:
I had the minified data in the description and now I have added the gzipped data also.